_install_guide/auth.md

```diff
@@ -90,6 +89,8 @@ security:
       email: email
       firstName: given_name
       lastName: family_name
+    userInfoRequirements: # Used to filter access by userInfo attributes
+      hd: armory.io # Accepts regular expression
```

Comment:
If the userInfo looks like this: https://developers.google.com/identity/protocols/OpenIDConnect#obtainuserinfo, then it's likely that some users might be interested in regex filtering by `email`, particularly because it appears that `hd` is only defined for G Suite users.

Basically, if the OpenID doc linked is applicable, maybe link to it here as well?

And for regex, may want an example because it needs to be in `/slashes/` if it is a regular expression.

Reply:
Good point. I'll add that in!